### DISCUSSION TITLE
Migrating from searchkit v2.4.4 to v4.10.1 issues related to "_msearch" instead of "_search"

### DISCUSSION THREAD

**Discussion by PrettyLegit** [searchkit/searchkit#1307] [Q&A]
I've been carefully following the tutorials from these docs:

- https://www.searchkit.co/docs/tutorials/with-nextjs
- https://www.searchkit.co/docs/api-documentation/searchkit'

I made sure that elastic search host server was running elasticsearch > 7.0... I'm on 7.1 of elastic search.
I am currently experiencing an issue where the old version of searchkit (v2.4.4) in the chrome network tab has the "host-url/_search" at the end and now on the new version of searchkit (with the `react-instantsearch`) i see "host-url/_msearch".

This is the error i got:
![image](https://github.com/searchkit/searchkit/assets/55465528/e5cc2454-10ab-4f56-971b-33907f37b801)

Any help is welcomed :)

I never had to set up an elastic search server before, and i mostly work with frontend implementation... but i am trying to upgrade this as much as i can without migration guides.

Thank you if you made it this far, u rock!

**Comment by joemcelroy** [comment]
hey! Are you calling elasticsearch directly from the browser? Does the network response look ok ( resolves 200, returns all results, no errors from ES)

**Reply by PrettyLegit** [reply]
I was getting 403 errors in my network, and i think the problem is how my project infrastructure is setup... im most likely going to now focus my attention on the aws service running elasticsearch that exposes the api that searchkit ends up using. hopefully that made sense... i've got more documentation to read now 😂.

Thanks for your support in the endeavors though, i will become more experienced after all this!
